ui/js/models/resultsets_store.js

```diff
@@ -143,11 +144,10 @@ treeherder.factory('ThResultSetStore', [
                     _.pick($location.search(), ["exclusion_profile", "visibility"]));
             }
             lastPolltime = Date.now();
-
             jobUpdatesPromise.then(function(jobList) {
+                jobList = _.flatten(jobList);
```

Comment:
you could use ``_flattenDeep()`` here, just to be safe.  Probably not any more expensive if it's only 2 levels.  But would ensure you don't leave extra levels if they somehow got in there.